packages/wme-ui/src/components/setup-card-accordion/setup-card-accordion.tsx

```diff
@@ -15,6 +15,7 @@ export interface SetupCardAccordionProps extends AccordionProps, Pick<SetupCardH
   header?: string,
   subHeader?: string;
   children: NonNullable<React.ReactNode>
+  toggleOpen?: (arg0?: string) => void;
```

Comment:
Appears to be a typo for `arg`.

Reply:
That's actually how Typescript wants anonymous arguments formatted. Since it's a "normal" function and not a React function, arguments aren't passed with names.